components/server/src/dump.rs

```diff
@@ -214,6 +222,53 @@ mod tests {
         let raft_engine = RaftLogEngine::new(raft_config);
         check_and_dump_raft_db(
             raftdb_path.to_str().unwrap(),
+            "",
+            &raft_engine,
+            Arc::new(Env::default()),
+            4,
+        );
+        assert(1, &raft_engine);
+        assert(5, &raft_engine);
+        assert(15, &raft_engine);
+    }
+
+    #[test]
+    fn test_dump_with_separate_wal() {
+        let data_path = tempfile::Builder::new().tempdir().unwrap().into_path();
+        let mut raftdb_path = data_path.clone();
+        let mut raftengine_path = data_path;
+        raftdb_path.push("raft");
+        raftengine_path.push("raft-engine");
+        let mut raftdb_wal_path = raftdb_path.clone();
+        raftdb_wal_path.push("wal");
```

Comment:
Should probably use more unique names, like `test_dump_with_separate_wal_raft`

Reply:
I'm not sure if it is necessary here. Each test here uses separate temp directory. I think it is safe to be consistent with the origin test.

Reply:
Let's at least differentiate from possible default paths, `data/raft` is the default raft path right now.

Reply:
I got you. That will be better.